src/charmory/tools/huggingface_eurosat_images.ipynb

```diff
@@ -0,0 +1,652 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datasets import Image as Im\n",
+    "import pyarrow as pa\n",
+    "import pandas as pd\n",
+    "from datasets import Dataset\n",
+    "from datasets import load_dataset\n",
+    "import datasets\n",
+    "from PIL import Image\n",
+    "import numpy as np\n",
+    "import jatic_toolbox"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Found cached dataset imagefolder (/home/chris/.cache/huggingface/datasets/tanganke___imagefolder/tanganke--EuroSAT-11e1b29eb44b9d39/0.0.0/37fbb85cc714a338bea574ac6c7d0b5be5aff46c1862c1989b20e0771199e93f)\n",
+      "Loading cached processed dataset at /home/chris/.cache/huggingface/datasets/tanganke___imagefolder/tanganke--EuroSAT-11e1b29eb44b9d39/0.0.0/37fbb85cc714a338bea574ac6c7d0b5be5aff46c1862c1989b20e0771199e93f/cache-49e74981f0392efc.arrow\n",
+      "Found cached dataset imagefolder (/home/chris/.cache/huggingface/datasets/tanganke___imagefolder/tanganke--EuroSAT-11e1b29eb44b9d39/0.0.0/37fbb85cc714a338bea574ac6c7d0b5be5aff46c1862c1989b20e0771199e93f)\n",
+      "Loading cached processed dataset at /home/chris/.cache/huggingface/datasets/tanganke___imagefolder/tanganke--EuroSAT-11e1b29eb44b9d39/0.0.0/37fbb85cc714a338bea574ac6c7d0b5be5aff46c1862c1989b20e0771199e93f/cache-f719f85a814ecbae.arrow\n",
+      "Found cached dataset imagefolder (/home/chris/.cache/huggingface/datasets/tanganke___imagefolder/tanganke--EuroSAT-11e1b29eb44b9d39/0.0.0/37fbb85cc714a338bea574ac6c7d0b5be5aff46c1862c1989b20e0771199e93f)\n",
+      "Loading cached processed dataset at /home/chris/.cache/huggingface/datasets/tanganke___imagefolder/tanganke--EuroSAT-11e1b29eb44b9d39/0.0.0/37fbb85cc714a338bea574ac6c7d0b5be5aff46c1862c1989b20e0771199e93f/cache-edf6901f5ee348c8.arrow\n"
+     ]
+    }
+   ],
+   "source": [
+    "def load_huggingface_dataset():\n",
+    "    validation_dataset = (jatic_toolbox.load_dataset)(\n",
+    "        provider=\"huggingface\",\n",
+    "        dataset_name=\"tanganke/EuroSAT\",\n",
+    "        task=\"image-classification\",\n",
+    "        split=\"validation\",\n",
+    "    )\n",
+    "    train_dataset = (jatic_toolbox.load_dataset)(\n",
+    "    provider=\"huggingface\",\n",
+    "    dataset_name=\"tanganke/EuroSAT\",\n",
+    "    task=\"image-classification\",\n",
+    "    split=\"train\",\n",
+    "    )\n",
+    "    test_dataset = (jatic_toolbox.load_dataset)(\n",
+    "    provider=\"huggingface\",\n",
+    "    dataset_name=\"tanganke/EuroSAT\",\n",
+    "    task=\"image-classification\",\n",
+    "    split=\"test\",\n",
```

Comment:
If you omit the `task` and `split` args to `jatic_toolbox.load_dataset`, you'll get the plain `DatasetDict` back. I wonder if pushing that up to the hub would retain the metadata to allow it to work with `jatic_toolbox.load_dataset`.

Reply:
I tried this but it still gave the same error with jatic_toolbox and the task not existing. I cleaned it up however and got rid of some useless lines.